docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+---
+version: '3.1'
+
+services:
+  postgres:
+    image: postgres:15
+    restart: always
+    environment:
+      - POSTGRES_DB=warehouse
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=password
+    ports:
+      - 5435:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER}"]
+      interval: 1s
+      timeout: 1s
+      retries: 5
+    volumes: 
+      - ./postgres-data:/var/lib/postgresql/data
+
+  liquibase:
+    depends_on: 
+      - postgres
+    image: liquibase/liquibase:latest
+    command: --url="jdbc:postgresql://postgres:5432/warehouse?user=postgres&password=password" --changelog-file="./changelog/changelog.yaml" update
```

Comment:
I'm not 100% happy with this, I wanted to use the `liquibase.properties` file loaded as a volume but the instance is having a hard time finding it and parsing it. I think it's not worth investing more time in this.